packages/@glimmer/benchmark-env/src/benchmark/if-helper.ts

```diff
@@ -1,18 +0,0 @@
-import { VMArguments } from '@glimmer/interfaces';
-import { Reference, createComputeRef, valueForRef } from '@glimmer/reference';
-
-export default function ifHelper({ positional }: VMArguments): Reference {
```

Comment:
This helper is no longer used, because `{{if}}` is now a keyword. Leaving it was an oversight, and since the signature changed I noticed because I had to update it.